<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab5SensorDatabase</class>
 <widget class="QWidget" name="Tab5SensorDatabase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QDateTimeEdit" name="pDateTimeEditFrom">
      <property name="date">
       <date>
        <year>2025</year>
        <month>7</month>
        <day>16</day>
       </date>
      </property>
      <property name="time">
       <time>
        <hour>9</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateTimeEdit" name="pDateTimeEditTo">
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2025</year>
        <month>12</month>
        <day>31</day>
       </datetime>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pPBsearchDB">
      <property name="text">
       <string>조회</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pPBdeleteDB">
      <property name="text">
       <string>삭제</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
    <item>
     <widget class="QTableWidget" name="pTBsensor">
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>50</number>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>날짜</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>조도</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,9">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="9,1">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pPBClearChart">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="pChartViewLayout"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
